uber/templates/registration/register.html

```diff
@@ -25,9 +23,14 @@
         $('.staffing').remove();
     };
     $(function () {
-        removeDonationRows();
-        removeVolunteerRows();
-        $('#bold-field-message').detach().prependTo('.form-horizontal');
+        {% if c.AT_THE_DOOR_BADGE_OPTS %}
+            maybeBold();
+            removeDonationRows();
+            removeVolunteerRows();
+            $('#bold-field-message').detach().prependTo('.form-horizontal');
+        {% else %}
+            $('.panel').empty().append('<h1>All badges are sold out!</h1>');
+        {% endif %}
```

Comment:
If I thought this had any serious chance of actually happening, I'd put more effort into styling it.  Or maybe I wouldn't and would instead spend my time rolling in money in the treasury room and making it rain.

In case it's not clear, the moment we run out of all badge types sold at the door, this page stops presenting and form and instead just displays a big header message saying we're all sold out.